### PR TITLE
Bgs main update

### PR DESCRIPTION
**Pull request by qmxp55**
I've updated the BGS MAIN target selection with two main changes.
1) We have relaxed our Fibre Magnitude Cut
2) We manually include the LSLGA galaxies as this are being rejected in DR8. I did not find the 'REF_CAT' parameter so I add it in '_prepare_optical_wise'.

I'll update the wiki in a bit.

### CODE REVIEW

File: py/desitarget/cuts.py [hunk 1]
```diff
@@ -1059,15 +1059,9 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
...
-        bgs &= rflux > 10**((22.5-19.5)/2.5)
-    elif targtype == 'faint':
-        bgs &= rflux > 10**((22.5-20.0)/2.5)
-        bgs &= rflux <= 10**((22.5-19.5)/2.5)
-    elif targtype == 'wise':
-        bgs &= rflux > 10**((22.5-20.0)/2.5)
-        bgs &= w1flux*gflux > (zflux*rflux)*10**(-0.2)
+    
+    #the LSLGA galaxies
+    L2 = ((refcat == 'L2') | (refcat == b'L2') | (refcat == 'L2 ') | (refcat == b'L2 '))
```
Comment (moustakas) [inline py/desitarget/cuts.py]: This bit of code will crash the integration tests / simulations. We need to capture the `if refcat is None` case.

Comment (qmxp55) [reply py/desitarget/cuts.py]: Hi John. Thanks. I've fixed that.

**Comment by qmxp55** [comment]
@moustakas: I've made a few changes and now it passes Travis tests.

File: py/desitarget/cuts.py [hunk 2]
```diff
@@ -1059,15 +1059,12 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
...
-        bgs &= rflux <= 10**((22.5-19.5)/2.5)
-    elif targtype == 'wise':
-        bgs &= rflux > 10**((22.5-20.0)/2.5)
-        bgs &= w1flux*gflux > (zflux*rflux)*10**(-0.2)
+    
+    #the LSLGA galaxies
+    if refcat is None:
+        L2 = fmc.copy()
+    else:
+        L2 = ((refcat == 'L2') | (refcat == b'L2') | (refcat == 'L2 ') | (refcat == b'L2 '))
```
[the remark this answers is not in the document]
Comment (qmxp55) [reply py/desitarget/cuts.py]: @geordie666: Done. Thanks for the help.

**Comment by geordie666** [comment]
@qmxp55: Thanks for the updates. Unless @moustakas has additional requests, I'll merge this later today.

**Pull request merged**